modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/VerticalBar/CustomVerticalBar.tsx

```diff
@@ -0,0 +1,80 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {VerticalBar} from '@clayui/core';
+import React, {ReactNode} from 'react';
+
+import './CustomVerticalBar.scss';
+
+interface CustomVerticalBarProps {
+	children: ReactNode;
+	defaultActive: string;
+	panelWidth?: number;
+	panelWidthMax?: number;
+	panelWidthMin?: number;
+	position: 'left' | 'right';
+	resize?: boolean;
+	triggerSideBarAnimation: boolean;
+	verticalBarItems: {
+		title: string;
+	}[];
+}
+
+export function CustomVerticalBar({
+	children,
+	defaultActive,
+	panelWidth = 1000,
+	panelWidthMax = 1200,
+	panelWidthMin = 400,
+	position,
+	resize = true,
+	triggerSideBarAnimation,
+	verticalBarItems,
+}: CustomVerticalBarProps) {
+	let verticalBarClassName;
+
+	if (triggerSideBarAnimation) {
```

Comment:
@igor-franca I believe this is not the updated commit creating this component. I remember we had replaced this if else with creating the `verticalBarClassName` as a template string receiving the position value inside it.